lddtree.sh

```diff
@@ -69,9 +68,11 @@ elf_specs_scanelf() {
 		sed -r 's: (LINUX|GNU)$: NONE:'
 }
 
+
+
 # functions for binutils backend
```

Comment:
what is the purpose of adding extra blank lines? I cannot see how that improves anything. It only obfuscates what the patch is supposed to do. The git commit message doesen't answer the question *why* this change is done, so the patch needs to be extra clear.